components/brave_wallet_ui/common/hooks/use-get-token-info.ts

```diff
@@ -58,10 +62,13 @@ export default function useGetTokenInfo(arg: Arg | typeof skipToken) {
       : undefined
   }, [arg, tokenInfoFromRpc])
 
+  const tokenInfoResult =
+    tokenInfoFromTokensList ?? nftFromRpc ?? tokenInfoFromRpc ?? undefined
+
   return {
     isVisible: tokenInfoFromTokensList?.visible ?? false,
-    tokenInfo:
-      tokenInfoFromTokensList ?? nftFromRpc ?? tokenInfoFromRpc ?? undefined,
-    isLoading: combinedTokensList.length === 0 || isFetching
+    tokenInfo: tokenInfoResult,
+    isLoading: combinedTokensList.length === 0 || isFetching,
+    isError: !isFetching && !tokenInfoResult && isError
```

Comment:
Can we change to `hasError`?

Reply:
We can, but i was trying to match the RTK signature: https://redux-toolkit.js.org/rtk-query/api/created-api/hooks#signature